packages/babel-plugin-transform-nullish-coalescing-operator/src/index.js

```diff
@@ -0,0 +1,33 @@
+import syntaxNullishCoalescingOperator from "babel-plugin-syntax-nullish-coalescing-operator";
+
+const OPERATOR = "??";
+
+export default function({ types: t }) {
+  return {
+    inherits: syntaxNullishCoalescingOperator,
+
+    visitor: {
+      LogicalExpression(path) {
+        const { node } = path;
+        if (node.operator !== OPERATOR) {
+          return;
+        }
+
+        const scope = path.scope.parent || path.scope;
+        const ref = scope.generateUidIdentifier("ref");
```

Comment:
We can use `#generateUidIdentifierBasedOnNode`, which'll create a better identifier name.